README.md

```diff
@@ -90,3 +90,23 @@ plt.show()
 
 <!-- Links -->
 [poetry]: https://python-poetry.org
+
+
+## `generate_hex_colors`
+
+This function generates the hex numbers for the colours extracted from a `matplotlib` colour map based on the the number of points of interest.
+The plot is there for reference to show what the swatches would look like if the function is used in  a Jupyter Notebook.
+
+```python
+import matplotlib.pyplot as plt
+import cosmoplots
+
+color_list = cosmoplots.generate_hex_colors(5, 'viridis', show_swatch=True, ascending=True)
+plt.savefig("./assets/color_test.png")
+
+# Print color_list to retrieve the hex numbers
+print(color_list)
```

Comment:
When I run these lines in my terminal the last line prints out
`['#fde725', '#5ec962', '#21918c', '#3b528b', '#440154']`
Maybe add the output or remove the last line in order to be consistent.

Reply:
Okay, I've added the output